### ISSUE TITLE
informational 

### ISSUE THREAD

**Issue by JonathanChiang** [IntelLabs/hpat#75]
Hi Ehsan, I noticed you included arrow in your build. I was wondering if HPAT's implementation is similar to Gandiva from Dremio. Though I believe the advantage of HPAT now is more focused on python whereas Gandiva is more focused on java. 

I was wondering if you had any opinions on zeromq. My dream is to utilize hpat, arrow and zeromq into a modeling serialization framework that's portable to different languages. 

https://www.dremio.com/announcing-gandiva-initiative-for-apache-arrow/

**Comment by ehsantn** [comment]
Yes, looks like Gandiva is compiling SQL expressions, which is semantically similar to some of HPAT.

Unfortunately, HPAT is focused on Python and making it language-agnostic is difficult.
